charts/factorio-server-charts/templates/deployment.yaml

```diff
@@ -102,7 +102,23 @@ spec:
             - name: account-data
               mountPath: /account
             {{- end }}
-      {{- end }}
+        {{- end }}
+        {{- if and (not .Values.factorioServer.generate_new_save) (.Values.factorioServer.import_save_file) (.Values.factorioServer.local_save_file_name) }}
+        - name: load-save-file
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy:  {{ .Values.image.pullPolicy }}
+          command:
+              - /bin/bash
+              - -ec
+              - |
+                ls -al 
+                mkdir /factorio/saves
+                echo "Copying saveFile from {{ .Values.factorioServer.local_save_file_name }} to /factorio/saves/{{ .Values.factorioServer.save_name }}.zip"
```

Comment:
could we just use the same save file name for both? I believe the save file name is important for loading the game on restart, you might end up generating a new save if not named properly. (i believe)